.github/workflows/build.sh

```diff
@@ -7,20 +7,27 @@ mydir=$(dirname $0)
 source ${mydir}/setup_env.sh
 
 print "======================================== Build"
-make -j8
+make -j8 || exit 10
 
 print "======================================== Install"
-make -j8 install
+make -j8 install || exit 11
 ls -R ${top}/install
 
 print "======================================== Verify build"
-ldd_result=$(ldd test/tester)
+ldd_result=$(ldd test/tester) || exit 12
 echo "${ldd_result}"
 
 # Verify that tester linked with cublas or rocblas as intended.
 if [ "${device}" = "gpu_nvidia" ]; then
-    echo "${ldd_result}" | grep cublas || exit 1
-fi
-if [ "${device}" = "gpu_amd" ]; then
-    echo "${ldd_result}" | grep rocblas || exit 1
+    echo "${ldd_result}" | grep cublas || exit 13
+
```

Comment:
Why is there an empty line, and below?

Reply:
The code got a little jammed together, so it was just easier to read.